.github/workflows/tests.yml

```diff
@@ -52,4 +67,5 @@ jobs:
       - name: Send screenshots to Percy
         run: npx percy exec -- jest --color --selectProjects "JavaScript component tests"
         env:
+          PERCY_PARTIAL_BUILD: 1
```

Comment:
The [`PERCY_PARTIAL_BUILD` environment variable](https://docs.percy.io/docs/environment-variables#optional) lets Percy know we won't always be sending the full 64 snapshots as we're intentionally holding back various "no-js" screenshots

It'll only flag changed screenshots for review, rather than missing ones

Reply:
Assuming we don't plan to send 'no-js' screenshots for those components again, can we not remove them instead?

Without this I'm assuming it'd get flagged in Percy but we could still approve and they'd be removed from the set of baseline screenshots used for comparison?

Reply:
I don't know enough about Percy really, what do you think we should do? Future thinking for `PERCY_PARTIAL_BUILD` was so we could enable `jest --changedSince=origin/main`

We might not need to send any screenshots if nothing has changed, but one for another PR

Shall I remove `PERCY_PARTIAL_BUILD` for now?

Reply:
I think so – I am making assumptions about how Percy works too but I imagine we'll be able to approve it with the 'no-js' screenshots missing and then they'll be removed from the base build and forgotten about.

Otherwise they'll be hanging around forever as 'things we used to send'.

Reply:
Thanks @36degrees I've removed `PERCY_PARTIAL_BUILD`

Reply:
Going to mark this resolved, mainly because it's gone 🚀